cves/CVE-2014-0117.yml

```diff
@@ -48,7 +48,12 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
+description: |
+  The HTTP Connection headers is left blank like 'Connection ;' so that the parser
+  will get an empty string and then it will skip over the semi colon and place it as the value of
+  the variable connection. This makes it so when the code for the proxy token is used it uses logic 
+  based on whitespaces and deliniated by commas and semi-colons. This becomes and inherent assumption 
+  that the input being passed is already sanitized and compatible.
```

Comment:
How might an attacker exploit this? Go a little further into what the risk is here.

Reply:
I'm not sure how much more I can go into it. "Connection ;" is and example of the exploit and then I explain how that exploit is parsed and how it affects the code further down the line. I am going to clean up the wording though